packages/archivematica-utils/src/index.ts

```diff
@@ -1,5 +1,12 @@
+import fetch from "node-fetch";
+import type { Response } from "node-fetch";
 import { logger } from "@stela/logger";
 
+const archivematicaHostUrl = process.env["ARCHIVEMATICA_HOST_URL"] ?? "";
```

Comment:
I know this is a common pattern in general and in our various code bases, but to clarify: if these are not set then that is a failure state, right?  If so, I think we would want to throw an error rather than silently use an empty string

In other projects I've used this little utility package for this purpose: https://www.npmjs.com/package/require-env-variable

It's (somewhat surprisingly, I think, since this is a common use case?) not used in a lot of places, but it's also such a straightforward package that I think that's OK: https://github.com/PabloSzx/require-env/blob/master/src/index.ts